src/v2/view-builder/views/consent/ConsentViewHeader.js

```diff
@@ -2,20 +2,21 @@ import { View, _ } from 'okta';
 
 import consentLogoHeaderTemplate from '../../../../views/shared/templates/consentLogoHeaderTemplate';
 
+const defaultLogo = '/img/logos/default.png';
 
 const ConsentViewHeader = View.extend({
   className: 'consent-title detail-row',
   template: consentLogoHeaderTemplate,
   getTemplateData: function () {
     const { appState } = this.options;
-    const { label, clientUri } =  appState.get('app');
+    const { currentFormName } = appState.get('currentFormName');
+    const { label, clientUri, logo } =  appState.get('app');
```

Comment:
@haishengwu-okta @pradeepdewda-okta I see these values can be set in the widgetrc, how does that work with appState in v2? I don't see these custom values overriding the AppState model. Is that something that's supposed to be a part of the SIW Customization project?

Reply:
> I see these values can be set in the widgetrc

Could you give example how those can be customized via widgetrc for v1?
I'm not aware.
If so, need to create ticket to the customization epic.

Reply:
You can set the `logo` in widgetrc to whatever path you want. That is different than the value in `appState.logo` which is coming from the server.

Reply:
`widgetrc.logo` is meant for identify view only (primary auth, passwordless auth, idp discovery  in v1)